tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -1433,6 +1433,23 @@ template <typename T> T wavePrefixProduct(T A, UINT) {
   return static_cast<T>(A * A);
 }
 
+template <typename T>
+struct Op<OpType::WavePrefixMatch, T, 1> : StrictValidation {};
+
+template <typename T> struct ExpectedBuilder<OpType::WavePrefixMatch, T> {
+  static std::vector<UINT> buildExpected(Op<OpType::WavePrefixMatch, T, 1> &,
+                                         const InputSets<T> &Inputs,
+                                         UINT WaveSize) {
+    DXASSERT_NOMSG(Inputs.size() == 1);
+
+    std::vector<UINT> Expected;
+    const size_t VectorSize = Inputs[0].size();
+    Expected.assign(VectorSize, 0);
```

Comment:
Aren't these operations expecting to only ever generate a single `uint4` result?  So it should be a vector of size 4, not the same size as the input vector.